docs/ZXing_barcode.md

```diff
@@ -0,0 +1,73 @@
+## ZXing Barcode Scanner Overview
+
+_Authored by [Brian Hayes](https://github.com/tomit4) 05/24/2023_
+
+### Introduction
+
+This document is meant to be a general overview for new users and developers for the PASS project's barcode scanner feature. At the time of this writing, this feature is specifically utilized to scan Driver's Licenses PDF417 barcodes in order to retrieve the data from the license, parse it into the appropriate .ttl (aka "Turtle") file format for storage on a Solid Client Server. While the development of this and other feature's of the PASS application is still currently ongoing, this document is meant to assist new user's of the PASS platform in becoming familiar with this feature and also to help new developers of the PASS project find where the logic for this feature lives within the code base.
+
+### For Users of PASS
+
+The current implementation of PDF417 barcode scanning of Driver's Licenses in PASS is imperfect, but functional. In order for the PDF417 barcode scanner to work appropriately, one must ensure that the barcode image is clear. Additionally the barcode image must be cropped reasonably so there are not any other aspects of the image that may interfere with the scanner. Finally, the barcode image must not be presented at a severe angle. Ideally the barcode image would appear very clearly, cropped specifically to show only the barcode, and be facing straight forward in the image. Here is an example of an ideal presentation of the barcode image:
+
+<p align="center">
+    <img src="https://github.com/codeforpdx/PASS/raw/Master/src/assets/demo_barcode.gif">
+</p>
+
+### For Developers
+
+**General Use**
+
+Currently PASS is in production, please see our [on-boarding](https://github.com/codeforpdx/PASS/blob/Master/CONTRIBUTING.md) to get a local version of PASS running. Alternatively you can visit our [live demo](https://codeforpdx.github.io/PASS/#/PASS/) to try it out. Once you have signed up with Solid, use your new username and password to log in. Navigate to the Forms section and you'll be presented with a series of options to upload documents.
+
+To test the barcode scanner specifically, head to the Upload Documents section. At the drop down form under "Select Document Type", select "Driver's License". You may optionally provide an expiration date for the document and also a description. Finally under the "File to upload" section, select the scanned image of the Driver's License PDF417 barcode from your computer. Click on the "Upload file" button and wait until you receive a short confirmation message below confirming your file has been uploaded.
```

Comment:
Instead of:

"At the drop down form under 'Select Document Type',..."

This:

"At the drop down input under 'Select Document Type',..."

Reply:
Switched to using` input` instead of `form`.